packages/lib/passport-interface/src/SubscriptionManager.ts

```diff
@@ -278,34 +293,66 @@ export class FeedSubscriptionManager {
     const responses: SubscriptionActions[] = [];
     this.resetError(subscription.id);
     try {
-      const authKey = await this.getAuthKeyForFeed(subscription);
+      const authKey = undefined; //await this.getAuthKeyForFeed(subscription);
```

Comment:
Is this unfinished or disabled for debug?

Reply:
I was thinking about whether to keep this in or remove it altogether. I've decided to keep it for now. It's a whole thing.